src/App.tsx

```diff
@@ -1,93 +1,52 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { TodoContext } from './components/TodosProvider/TodosProvider';
+import { Filter } from './types/Filter';
+import { Footer } from './components/Footer/Footer';
 
-export const App: React.FC = () => {
-  return (
-    <div className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+function prepareTodos(todos: Todo[], filter: Filter): Todo[] {
+  let todosCopy = [...todos];
```

Comment:
why use let in this case ?
Try to write this function without let. 
If you can write constant instead let use it prefer.
let worsens the readability and perception of the code during scaling